src/plugins/lightbox/_base.scss

```diff
@@ -11,6 +11,10 @@
 	outline-offset: -2px;
 }
 
+%lightbox-modal-hide {
+	visibility: hidden!important;
```

Comment:
If there is a rule that applies, it's just how it is formatted in the rest of the repo https://github.com/wet-boew/wet-boew/search?utf8=%E2%9C%93&q=%21important